python/cugraph/cugraph/structure/graph_classes.py

```diff
@@ -81,7 +81,13 @@ def __init__(self, m_graph=None, directed=False):
                 )
 
     def __getattr__(self, name):
-        # accessing `_Impl` will lead to recursion
+        """
+        __getattr__() is called automatically by python when an attribute does not
+        exist. Since this class is attempting to hide the internal `_Impl` object,
+        which is intended to contain many of the attributes needed by this class,
+        __getattr__ is used to "pass through" attribute access to _Impl and make it
+        appear as if the _Impl attributes are contained in this class.
+        """
         # so avoiding that
```

Comment:
oops, probably want to delete this line now.

Reply:
Done